brightness-mode-explainer.md

```diff
@@ -0,0 +1,128 @@
+# Design Document: the need to control screen brightness
+
+## Introduction
+
+Since 2018, the working group has received [significant requests](https://github.com/w3c/screen-wake-lock/issues/129) from web developers surrounding the need to increase the brightness of a device's screen.
+
+For developers, it would help address the following use cases:
+
+- a user scans QR/barcode under various lighting conditions, in which case boosting the screen brightness creates more contrast.
+  - this is already possible on native mobile apps such as [Starbucks'](https://play.google.com/store/apps/details?id=com.starbucks.mobilecard), [the UK Home Office's Document Check app](https://play.google.com/store/apps/details?id=uk.gov.HomeOffice.ho1) and the [Dutch government's CoronaCheck app](https://play.google.com/store/apps/details?id=nl.rijksoverheid.ctr.holder).
+- increasing the screen brightness can help illuminate a user's facial features, improving the viability of certain application that rely on the the front facing camera (e.g., [make up mirror](https://play.google.com/store/apps/details?id=mmapps.mirror.pro&hl=en&gl=US) style apps, and applications that can scan people's facial features to perform some particular action)
+
+## Open design issues
+
+The following issues remain open for discussion:
+  - should UAs be free to limit maximum brightness level and/or do other things like increase contrast?
+  - Should this be mobile first/only? (i.e. not worry about external displays).
+  - Should this trigger "`prefers-contrast: more`" in [CSS Media Queries](https://drafts.csswg.org/mediaqueries-5/#prefers-contrast)?
+- From [https://github.com/w3c/devicesensors-wg/issues/51](https://github.com/w3c/devicesensors-wg/issues/51)
+  - Brightness levels
+    - How bright is too bright? Consider 100% brightness with HDR displays, for example.
+    - Take a discrete or continuous value?
+    - Related to whether script should be allowed to reduce brightness.
+  - Permission model
+    - Would it require user gesture (request but not consume it)?
+  - While brightness changes
+    - What if users change the brightness level while the lock is held?
+  - When dropping screen brightness request
+      - Does the UA have to restore the previous brightness level?
+        - What about external displays? Do UAs need to keep track of levels for each display?
+      - Should script be allowed to "hold the lock" indefinitely?
+
+## Goals
+
+- Provide the ability to request an increase to the display's brightness. This could be a UA-controlled maximum (which *could* be the maximum display brightness), either indefinitely or for a period of time.
+- Provide the ability to release the request so that the device's brightness returns to its pre-request value (i.e. hand back control to OS).
```

Comment:
```suggestion
- Provide the ability to release the request so that the device's brightness returns to its pre-request value (i.e., hand back control to OS).
```